[CTOR-1282]-Refresh hardware-storage-emc-vplex-restapi doc page (#3982)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/hardware-storage-emc-vplex-restapi.md

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

## Connector dependencies

> The following monitoring connectors will be installed when you install the **EMC Vplex** connector through the

Reviewer: ```suggestion
The following monitoring connectors will be installed when you install the **EMC Vplex Rest API** connector through the
```

**Configuration > Monitoring Connector Manager** menu:
* [Base Pack](./base-generic.md)

## Pack assets

### Templates

> The Monitoring Connector **EMC Vplex** brings a host template:

Reviewer: ```suggestion
The Monitoring Connector **EMC Vplex Rest API** brings a host template:
```

</TabItem>
</Tabs>

3. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The service appears in the list of services, and on the **Resources Status** page. The command that is sent by the connector is displayed in the details panel of the service: it shows the values of the macros.

## How to check in the CLI that the configuration is OK and what are the main options for?

Once the plugin is installed, log into your Centreon poller's CLI using the
**centreon-engine** user account (`su - centreon-engine`). Test that the connector 
is able to monitor a resource using a command like this one (replace the sample values by yours):

```bash
/usr/lib/centreon/plugins/centreon_emc_vplex.pl \
	--plugin=storage::emc::vplex::restapi::plugin \
	--mode=storage-volumes \
	--hostname='10.0.0.1' \
	--custommode='apiv1' \
	--proto='https' \
	--port='443' \
	--vplex-username='' \

> 	--vplex-password=''  \

Reviewer 2: ```suggestion
        --vplex-username='XXXX' \
	--vplex-password='XXXX'  \
```

> The Centreon Pack supports the Rest APIv1 and APIv2. Services **Fans** and **Psus** are not supported by the Rest APIv2.

Reviewer: ```suggestion
The connector supports the Rest APIv1 and APIv2. The **Fans** and **Psus** services are not supported by the Rest APIv2.
```

## Installing the monitoring connector

### Pack

1. If the platform uses an *online* license, you can skip the package installation
instruction below as it is not required to have the connector displayed within the
**Configuration > Monitoring Connector Manager** menu.
If the platform uses an *offline* license, install the package on the **central server**
with the command corresponding to the operating system's package manager:

<Tabs groupId="sync">
<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">

```bash
dnf install centreon-pack-hardware-storage-emc-vplex-restapi
```

</TabItem>
<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">

```bash
dnf install centreon-pack-hardware-storage-emc-vplex-restapi
```

</TabItem>
<TabItem value="Debian 11 & 12" label="Debian 11 & 12">

```bash
apt install centreon-pack-hardware-storage-emc-vplex-restapi
```

</TabItem>
<TabItem value="CentOS 7" label="CentOS 7">

```bash
yum install centreon-pack-hardware-storage-emc-vplex-restapi
```

</TabItem>
</Tabs>

> 2. Whatever the license type (*online* or *offline*), install the **EMC Vplex** connector through

Reviewer: ```suggestion
2. Whatever the license type (*online* or *offline*), install the **EMC Vplex Rest API** connector through
```

</TabItem>
</Tabs>

3. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The service appears in the list of services, and on the **Resources Status** page. The command that is sent by the connector is displayed in the details panel of the service: it shows the values of the macros.

## How to check in the CLI that the configuration is OK and what are the main options for?

Once the plugin is installed, log into your Centreon poller's CLI using the
**centreon-engine** user account (`su - centreon-engine`). Test that the connector 
is able to monitor a resource using a command like this one (replace the sample values by yours):

```bash
/usr/lib/centreon/plugins/centreon_emc_vplex.pl \
	--plugin=storage::emc::vplex::restapi::plugin \
	--mode=storage-volumes \
	--hostname='10.0.0.1' \
	--custommode='apiv1' \
	--proto='https' \
	--port='443' \
	--vplex-username='' \

> 	--vplex-password=''  \

Reviewer: same as FR
